securedrop_client/gui/widgets.py

```diff
@@ -1972,96 +2001,137 @@ def _update_text(self, source_id: str, message_uuid: str, text: str) -> None:
         """
         if message_uuid == self.uuid:
             self.message.setText(text)
+        self.set_normal_styles()
```

Comment:
this should be tabbed in so that you only reset back to the normal styling if the message uuid matches

Reply:
good catch, thanks